nbclassic/shim.py

```diff
@@ -229,11 +234,14 @@ def shim_config_from_notebook_to_jupyter_server(self, config):
         # 3. Handle ExtensionApp traits.
         warning_msg = None
         for trait_name, trait_value in extapp_config.items():
+            if trait_name in IGNORED_TRAITS: continue
```

Comment:
This line seems contrary to what you added below and I'm wondering if it should be deleted.
```suggestion
```

Reply:
Good catch! Thanks, @kevin-bates!